### DISCUSSION TITLE
Add a Telegram icon to the icon extension?

### DISCUSSION THREAD

**Discussion by glow12121** [datenstrom/community#557] [Ask a question] [answered]
Hi,

[An icon for the popular messenger Telegram](https://fontawesome.com/icons/telegram-plane?style=brands) seems to be absent from the [corresponding extension](https://github.com/annaesvensson/yellow-fontawesome) for Yellow. Could you please add it?

**Comment by markseuffert** [comment, marked as answer]
I updated the extension. Now there's `fa-telegram` and `fa-paper-plane`, please try them out.

**Comment by glow12121** [comment]
Mark, you are a star! It works just fine.
